InventoryIndexer/etc/queue_topology.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework-message-queue:etc/topology.xsd">
+    <exchange name="magento-db" type="topic" connection="db">
+        <binding id="InventoryReservationPlaceDb" topic="inventory.reservation.place" destinationType="queue" destination="inventory.reservation.place"/>
```

Comment:
InventoryReservationPlaceDb - it's not exactly what we will be doing here. We are going to update Salability status. 
Reservations are being created in the main thread